src/lib/application.py

```diff
@@ -296,15 +300,124 @@ def is_analysis_complete(self):
 
 class CodeReadyStudio(Application):
     """
-    CodeReadyStudio class for managing RH Code Ready Studio application
+    Class for managing RH Code Ready Studio application
     """
 
     pass
 
 
 class Eclipse(Application):
     """
-    Eclipse class for managing Eclipse application
+    Class for managing Eclipse application
     """
 
     pass
+
+
+class VisualStudioCode(Application):
+    """
+    Class for managing VSCode application
+    """
+
+    def close_ide(self):
+        """
+        Closes the IDE
+        """
+        self.press_keys("ctrl", "q")
+
+    def is_open_mta_perspective(self):
+        """
+        Checks if MTA perspective is already opened in VS Code
+
+        Returns:
+            (bool): True or False
+        """
+        try:
+            self.wait_find_element(locator_type="image", locator="mta_config_active.png")
+            return True
+        except Exception as exc:
+            logging.debug(
+                "An error occured while finding \
+                MTA perspective tab ! {}".format(
+                    str(exc),
+                ),
+            )
+            if "No matches found" in str(exc):
+                return False
+            else:
+                raise Exception(exc)
+
+    def open_mta_perspective(self):
+        """
+        Opens MTA perspective in VSCode IDE
+        """
+        if self.is_open_mta_perspective():
+            logging.info("MTA perspective is already opened !")
+            return
+        else:
+            """
+            Click on the MTA icon in left sidebar
+            """
+            self.click_element(locator_type="image", locator="mta_config_view.png")
+
+    def run_simple_analysis(self, project, packages=[]):
+        """
+        Runs analysis by adding the project and/or packages passed as argument
+
+        Args:
+            project (str): Full name of project to be analysed
+            packages (list): List of packages to be added to analysis
+
+        Returns:
+            None
+        """
+        """
+        Click on Run MTA Configuration icon -> Click new config
```

Comment:
We can add these as steps in one single comment



     """
        Runs analysis by adding the project and/or packages passed as argument
        Args:
            project (str): Full name of project to be analysed
            packages (list): List of packages to be added to analysis
        Returns:
            None
        Steps:
        1) Click on Run MTA Configuration icon 
        2) Click new config